import-export-cli/cmd/getAPIProductRevisions.go

```diff
@@ -0,0 +1,91 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+	"github.com/wso2/product-apim-tooling/import-export-cli/credentials"
+	"github.com/wso2/product-apim-tooling/import-export-cli/impl"
+
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var getRevisionsAPIProductName string
+var getRevisionsAPIProductProvider string
+var getAPIProductRevisionsCmdEnvironment string
+var getAPIProductRevisionsCmdFormat string
+var getAPIProductRevisionsCmdQuery string
+
+// GetAPIProductRevisionsCmd related info
+const GetAPIProductRevisionsCmdLiteral = "api-product-revisions"
+const GetAPIProductRevisionsCmdShortDesc = "Display a list of Revisions for the API"
+
+const GetAPIProductRevisionsCmdLongDesc = `Display a list of Revisions available for the API in the environment specified`
```

Comment:
Shall we change this?
```suggestion
const GetAPIProductRevisionsCmdLongDesc = `Display a list of Revisions available for the API Product in the environment specified`
```